
{% extends "base.html" %}
{% block body %}
<h1>The team</h1>
  <table class="colleagues">
  <tr>
  <td><div class="motto">
    <h3><strong>Guillaume Filion</strong>, team leader</h3>
    <p><a href="/static/misc/CV_Guillaume_Filion.pdf"><i>curriculum
    vitae</i></a></p>
    <p>The greatest thing about being a biologist today is that
    we can finally address one of the most fundamental questions
    about life: what is biological meaning? If we understand how
    information is encoded in DNA, we are far from understanding
    the meaning of that information. Our team works on regulatory
    sequences, their evolution and their activity in different
    contexts. In other terms, we want to understand how non coding
    DNA is decoded.
    </p>
  </div></td>
  <td class="picture"><img src="/static/images/Guillaume_cropped.jpg"></td>
  </tr>

  <tr>
  <td><div class="motto">
    <h3><strong>Heng-Chang Chen</strong>, post-doc</h3>
    <p>看不見，可是你依舊存在 is a famous article written by
    <a href="http://en.wikipedia.org/wiki/Chiang_Ching-kuo">Ching-Kuo
    Chiang</a> in Taiwan, which means “You exist even though you are
    invisible”. The same is true about latent HIV-1, lying dormant in
    various reservoirs, awaiting to revive.
    Nowadays, latency has become the main roadblock to the development of
    a cure for HIV-1. My project is to map latent HIV-1 integrations in
    a high throughput setup. I call the output of this technology
    “marauder maps”, because they expose the position of every silent HIV-1
    integration. I am also interested in data visualisation and
    scientific illustration.</p>
    <p>You can visit my <a href="http://www.dzdm.co.uk">personal page</a>
    to find out more about me.
    </p>
  </div></td>
  <td class="picture"><img src="/static/images/HengChang_cropped.jpg"></td>
  </tr>


  <tr>
  <td><div class="motto">
    <h3><strong>Marc Corrales</strong>, PhD student</h3>
    <p>With the advent of genome-wide, high-troughput technologies, we can
    for the first time look at genome architecture and gene regulation
    without losing the forest for the trees.
    My research interest is trying to understand how the text
    works in context, <i>i.e</i>  how the 98% non protein-coding DNA in our
    cells is able to regulate the 2% protein-coding DNA to produce the most
    dynamic and complex system we know so far. Currently I am involved in
    studing the role of Black chromatin in silencing gene expression
    with the TRiP technology developed in our laboratory.</p>
  </div></td>
  <td class="picture"><img src="/static/images/Marc_cropped.jpg"></td>
  </tr>

  <tr>
  <td><div class="motto">
    <h3><strong>Arantxa Rosado</strong>, technician</h3>
    <p>There are many layers of information encoded in the genome. The DNA
sequence, the epigenetic marks and the chromatin context are the main
players of the gene regulation machinery. To investigate the role of
each of these factors in the process  we are developing a new
technology TRiP (Thousand of Reporters in Parallel).
Using TRiP we can monitor the expression and localization of reporters
integrated at different loci in the genome. Would the chromatin invade
the reporter or would it preserve its own identity? This questions and
others will be hopefully answer soon :-)</p>
  </div></td>
  <td class="picture"><img src="/static/images/Arantxa_cropped.jpg"></td>
  </tr>

  <tr>
  <td><div class="motto">
    <h3><strong>Pol Cuscó</strong>, PhD student</h3>
    <p>Coming soon...</p>
  </div></td>
  <td class="picture"><img src="/static/images/Pol_cropped.jpg"></td>
  </tr>

  <tr>
  <td><div class="motto">
    <h3><strong>Victoria Pokusaeva</strong>, technician</h3>
    <p>Coming soon...</p>
  </div></td>
  <td class="picture"><img src="/static/images/Vika_cropped.jpg"></td>
  </tr>

  <tr>
  <td><div class="motto">
    <h3><strong>Eduard Valera</strong>, bioinformatician</h3>
    <p>Since the first draft of the human genome was published, we
    realized that our genes comprise only 2% of the full sequence. We
    need to understand the remaining 98% to explain crucial cell
    mechanisms such as cell differentiation or transcription regulation.
    Most of this non-coding sequences follow complex logics that lie
    beyond human understanding. Besides, there seems to be relevant
    information encoded in the structural and spatial conformation of
    the genomes. My role in the lab is to design algorithms and
    computational methods that help us understand how these mechanisms
    are regulated.</p>
  </div></td>
  <td class="picture"><img src="/static/images/Eduard_cropped.jpg"></td>
  </tr>

  <tr>
  <td><div class="motto">
    <h3><strong>Catalina Romero</strong>, post-doc</h3>
    <p>Coming soon...</p>
  </div></td>
  <td class="picture"><img src="/static/images/Catalina_cropped.jpg"></td>
  </tr>

  <tr>
  <td><div class="motto">
    <h3><strong>Ruggero Cortini</strong>, post-doc</h3>
    <p>The complexity and beauty of life has always fascinated me. As
    a physicist, the fine organization of living matter - spanning all
    scales from molecules to organisms to ecosystems - seems to me out
    of the reach of human comprehension. However, we now for the first
    time have the opportunity to understand more about the biological
    reality, having access to more and more detailed information on
    every level of its organisation. I am interested in understanding
    how basic physical principles, from the atomic scale to the scale
    of the cell nucleus, translate into biological meaning. I study
    the effect of physical interactions on the conformation of DNA
    and proteins in the cell, seeking to make a connection to gene
    expression and regulation.</p>
  </div></td>
  <td class="picture"><img src="/static/images/Ruggero_cropped.jpg"></td>
  </tr>
</table>

<h2>Alumni</h2>

<table id="alumni" class="colleagues">
  <tr>
  <td><div class="motto">
    <h3><strong>Ricky Lim</strong></h3>
    <h4>2012-13</h4>
    <p>“The twentieth century was the century of physics and the
    twenty-first century will be the century of biology and technology”.
    Whether or not you believe in it. Thinking about this inspired me:
    I would challenge myself to integrate my biological perspectives with
    computational approaches.</p>
  </div></td>
  <td class="picture"><img src="/static/images/Ricky_cropped.jpg"></td>
  </tr>

  <tr>
  <td><div class="motto">
    <h3><strong>Catarina De Oliveira</strong></h3>
    <h4>2012-14</h4>
    <p>Transcription regulation requires coordination of transcription
    machinery and transcription factors at promoter sites. Recently,
    some evidence suggested that compartmentalization is also necessary
    for genome regulation, but how chromatin context and structure
    influences this is still unclear.
    I am particularly interested in hormone-responsive elements and
    how their spatial distribution affects activation of this elements
    and consequently the transcription of downstream genes.</p>
  </div></td>
  <td class="picture"><img src="/static/images/Catarina_cropped.jpg"></td>
  </tr>

  <tr>
  <td><div class="motto">
    <h3><strong>Olivera Vujatovic</strong></h3>
    <h4>2012-15</h4>
    <p>It is very challenging working on recently available genome-wide
    data about chromatin proteins distribution and genome sequences
    in different species, including non-coding elements. They are
    giving great opportunities to understand better interplay between
    DNA and proteins that bind them, how they regulate transcription
    and an insight into evolutionary paths that shaped them. With the
    help of statistics one can process this big amount of the data
    and observe general concepts that might explain questions about
    genome functioning.</p>
  </div></td>
  <td class="picture"><img src="/static/images/Olivera_cropped.jpg"></td>
  </tr>
</table>



{% endblock %}
{% block section %}<div id="section" title="#people"></div>{% endblock %}
